### PR TITLE
BWDB-13620 api spec for importVoiceTn and removeImportedVoiceTn orders

### PR DESCRIPTION
**Pull request by xxlaykxx**
POST GET /accounts/{accountId}/importTnOrders/voice
GET /accounts/{accountId}/importTnOrders/voice/{id}
GET /accounts/{accountId}/importTnOrders/voice/{id}/history

POST GET /accounts/{accountId}/removeImportedTnOrders/voice
GET /accounts/{accountId}/removeImportedTnOrders/voice/{id}
GET /accounts/{accountId}/removeImportedTnOrders/voice/{id}/history
Снимок экрана 2022-07-08 в 18 46 54
Снимок экрана 2022-07-08 в 18 47 16

### CODE REVIEW

File: site/specs-temp/numbers.json [hunk 1]
```diff
@@ -7515,6 +7515,292 @@
         }
       }
     },
+    "/accounts/{accountId}/importTnOrders/voice": {
+      "get": {
+        "tags": [
+          "/accounts"
+        ],
+        "description": "Retrieves the ImportVoiceTnOrder requests for the given account ID. A maximum of 1,000 orders can be retrieved per request. If no date range or specific query parameter (marked by <b class=\"required\">*</b> below) is provided, the order results will be limited to the last two years.\n",
```
Comment (bpateldx) [inline site/specs-temp/numbers.json:7523]: 1. Why send 1000 be default for up to 2 years? Why not the requestor at least pass one of the criteria and then limit to 1000?
2. How do the requestor request next 1000 and so on or for that use `history` API?
3. @ajrice6713 Is the `/` needed for specific reason in tag names?

File: site/specs-temp/numbers.json [hunk 2]
```diff
@@ -7515,6 +7515,292 @@
...
+        "summary": "List Import Voice Tn orders",
+        "operationId": "ReadImportVoiceTnOrders",
+        "parameters": [
+          {
+            "name": "accountId",
```
Comment (bpateldx) [inline site/specs-temp/numbers.json:7528]: `accountId`, `orderId` and `status` should be defined as reusable `components`. The `status` should be named accordingly and defined as `enum`

File: site/specs-temp/numbers.json [hunk 3]
```diff
@@ -7515,6 +7515,292 @@
         }
       }
     },
+    "/accounts/{accountId}/importTnOrders/voice": {
```
Comment (bpateldx) [inline site/specs-temp/numbers.json:7518]: Any rationale behind adding the `voice` suffix? It does not seem to be needed

File: site/specs-temp/numbers.json [hunk 4]
```diff
@@ -30001,6 +30566,295 @@
...
+              "TelephoneNumbers": {
+                "type": "array",
+                "items": {
+                  "$ref": "#/components/schemas/TelephoneNumber"
+                }
+              },
+              "Errors": {
+                "type": "array",
+                "items": {
+                  "$ref": "#/components/schemas/TelephoneNumberGroupError"
```
Comment (bpateldx) [inline site/specs-temp/numbers.json:30852]: Where would the requestor see problematic `tns` not being successfully imported? Inside `TelephoneNumberGroupError`?

**Review by bpateldx — CHANGES_REQUESTED** [review]
Please review comments and do what's necessary

File: site/specs-temp/numbers.json (hunk 3 above)
Comment (ajrice6713) [reply site/specs-temp/numbers.json:7518]: Its an added service/new endpoint that they have to stuff behind the existing /importTnOrders endpoint - it is a new endpoint + required

File: site/specs-temp/numbers.json (hunk 1 above)
Comment (ajrice6713) [reply site/specs-temp/numbers.json:7523]: The `/` shows in the documentation sidebar 

we have work scheduled later to go clean up those tags - its outside the scope of this PR to fix them now

File: site/specs-temp/numbers.json (hunk 4 above)
Comment (xxlaykxx) [reply site/specs-temp/numbers.json:30852]: yes

File: site/specs-temp/numbers.json (hunk 1 above)
Comment (xxlaykxx) [reply site/specs-temp/numbers.json:7523]: 1, 2 it's how our generic history api works. we couldn't change it only for this order type

File: site/specs-temp/numbers.json (hunk 2 above)
Comment (xxlaykxx) [reply site/specs-temp/numbers.json:7528]: fixed

Comment (xxlaykxx) [reply site/specs-temp/numbers.json:7528]: @bpateldx plz re-review

File: site/specs-temp/numbers.json (hunk 1 above)
Comment (bpateldx) [reply site/specs-temp/numbers.json:7523]: @xxlaykxx I'm fine for 1. On 2, can you explain how the client will get next 1000 results and so on? i.e. is support for pagination needed?

**Review by bpateldx — APPROVED** [review]

**Review by ajrice6713 — APPROVED** [review]

**Pull request merged**